pkg/inspector/image-inspector_test.go

```diff
@@ -165,6 +166,43 @@ func TestGetAuthConfigs(t *testing.T) {
 	}
 }
 
+func Test_decodeDockerResponse(t *testing.T) {
+	no_error_input := "{\"Status\": \"fine\"}"
+	one_error := "{\"Status\": \"fine\"}{\"Error\": \"Oops\"}{\"Status\": \"fine\"}"
+	tests := map[string]struct {
+		readerInput    string
+		expectedErrors bool
+	}{
+		"no error": {readerInput: no_error_input, expectedErrors: false},
+		"error":    {readerInput: one_error, expectedErrors: true},
```

Comment:
test for decode error too.  Also, please check to make sure you're getting the message you think you are.  I had an issue with some of my code where the test errored correctly but it was never hitting the code I thought it was...oops :smile: